pages/common/ugrep.md

```diff
@@ -0,0 +1,36 @@
+# ugrep
+
+> Ultra fast search tool with query TUI.
+> More information: <https://github.com/Genivia/ugrep> and `ugrep --help {{option_or_word}}`.
```

Comment:
```suggestion
> More information: <https://github.com/Genivia/ugrep>`.
```

This doesn't fit our syntax unfortunately, however it could be added as an example.